docs/overview/sapi.rst

```diff
@@ -33,191 +33,211 @@ and scheduling for D-Wave solvers. Problem submission through SAPI requires the
+                -   You can override this configuration by setting a solver in an environment
+                    variable or explicitly in your code.
+            *   -   :std:doc:`Environment variables <oceandocs:docs_cloud/reference/configuration>`
+                -   Configure :code:`DWAVE_API_SOLVER`. For example,
+                    :code:`export DWAVE_API_SOLVER='{"num_qubits__gt": 2000}'` in
+                    a Unix shell.
+                -   You can override this configuration by selecting a solver explicitly
+                    in your code.
+            *   -   Explicit parameter
+                -   You can set your solver selection directly in your code; for example,
+                    :code:`sampler = DWaveSampler(solver=dict(topology__type='pegasus'))`
+                -
+
+*   **Region/Endpoint** (default: North American URL)
+
+    A URL for a region's remote resources.
+
+    By default, Ocean connects to North American (region :code:`na-west-1`) Leap
+    quantum cloud resources at URL
+    :code:`https://na-west-1.cloud.dwavesys.com/sapi/v2/`.
+
+    .. dropdown:: Finding Supported Regions and Endpoints
+
+        The :ref:`table_regions_viewing` table shows various options for viewing
+        available regions and their URLs.
+
+        .. list-table:: Viewing Available Regions and Endpoints
+            :name: table_regions_viewing
+            :header-rows: 1
+
+            *   -   Where
+                -   How
+                -   Usage Notes
+            *   -   `Leap <https://cloud.dwavesys.com/leap/>`_ dashboard
+                -   Log in to your Leap account
+                -   Solvers available to your account are grouped by region.
+            *   -   :ref:`dwave-cloud-client <sdk_index_cloud>`
+                -   Use the :ref:`Interactive CLI <dwave_cli>` or
+                    :meth:`~dwave.cloud.client.Client.get_regions`
+                -
+
+        Example: using :doc:`dwave-cloud-client </docs_cloud/sdk_index>` to query
+        supported regions.
+
+            >>> from dwave.cloud import Client
+            >>> with Client.from_config() as client:          # doctest: +SKIP
+            ...    regions = client.get_regions()
+            ...    for code, info in regions.items():
+            ...        print(f"{info['name']} ({code}): {info['endpoint']}")
+            North America (na-west-1): https://na-west-1.cloud.dwavesys.com/sapi/v2/
+            Europe (eu-central-1): https://eu-central-1.cloud.dwavesys.com/sapi/v2/
+
+        .. note:: Users of on-premises systems should request the SAPI endpoint from
+            system administrator.
 
         .. TODO (need info): For non-Ocean clients, you can retrieve a list of
-           supported regions by sending an :code:`HTTP GET` request to the
-           :code:`<Metadata URL>/regions/` endpoint.
+            supported regions by sending an :code:`HTTP GET` request to the
+            :code:`<Metadata URL>/regions/` endpoint.
```

Comment:
Metadata API endpoint in question is: `https://cloud.dwavesys.com/metadata/v1/regions`.